juicebox_cli/cli.py

```diff
@@ -87,7 +87,7 @@ def upload(ctx, client, env, app, job, netrc, files):
         logger.debug(message)
         click.echo(click.style(message, fg='red'))
         ctx.abort()
-    except AuthenticationError as exc_info:
+    except Exception as exc_info:
```

Comment:
any particular reason we're using the broad Exception type?

Reply:
yeah, since this is a top-level script I wanted to be able to print out the "nicer" error message regardless of the type. We were already doing it for AuthenticationError, and instead of adding a bigger list of all the different exceptions we'd want to handle this way, I just generalized it.